pkg/kexec/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:145f062a40639b6c65efa36bed1c5614b873be52
+FROM ${EVE_BUILDER_IMAGE} AS build
+
+ENV BUILD_PKGS patch curl make gcc perl util-linux-dev git mtools linux-headers musl-dev xz-dev elfutils-dev libbz2
+ENV PKGS xz-libs elfutils-dev libbz2
```

Comment:
I can see that tests are failing with `failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: nsenter: unrecognized option: a\nBusyBox v1.35.0...`. The reason is in build process of rootfs, when linuxkit handle init section, it append layers on top of each one. So with kexec layer on top of dom0-ztools we can see that nsenter come from busybox, not from util-linux (more precise, util-linux-misc). There are two possible ways to solve the issue:

- reorder rootfs.yml.in and place kexec before dom0-ztools
- add util-linux here in PKGS

Reply:
Is it anything special in these tests? How can I reproduce the same failure? Would like to run the same stuff locally, please guide me. Thanks.

Reply:
You can try to run for example `eve exec ip a` from console.

Reply:
Or you can run `chroot /hostfs` and `eve exec pillar ip a` from ssh. We [use](https://github.com/lf-edge/eve/blob/c61a93416772a3735869e9ff4553c1388f70291b/pkg/pillar/containerd/oci.go#L36) `eve exec` from rootfs in [hooks](https://github.com/lf-edge/eve/blob/c61a93416772a3735869e9ff4553c1388f70291b/pkg/pillar/containerd/oci.go#L267-L291) when we run containers (on node without hardware virtualization support, for example on GH VM in Eden tests).

Reply:
Thanks, Petr for pointing that out. I am able to reproduce and fix the problem by providing the util-linux to the kexec.

Am I correct that the root cause is that since we have a sandwich of these layers the last one (kexec) does not provide the original nsenter tool to jump into the namespace, so that nsenter from busybox is used, which apparently does fully implement the behavior of the original nsenter tool?

Reply:
Exactly. We try to avoid copy of full alpine rootfs and copy only required files after dom0-ztools. This looks rather strange...

Reply:
Argh, this is so unobvious.  As the whole linuxkit approach I would say, ok, refrain from grunting :)